conf/app.routes

```diff
@@ -12,6 +12,8 @@ POST       /submissions/update                                      uk.gov.hmrc.
 
 POST       /create/return                                           uk.gov.hmrc.formpproxy.sdlt.controllers.returns.ReturnsController.createSDLTReturn()
 POST       /retrieve-return                                         uk.gov.hmrc.formpproxy.sdlt.controllers.returns.ReturnsController.getSDLTReturn()
+POST       /returns                                                 uk.gov.hmrc.formpproxy.sdlt.controllers.returns.ReturnsController.getSDLTReturns()
```

Comment:
can we change this to **POST /get/returns**

we are aligning with this convention for routes:

`POST /delete/predefined-agent`
`POST /update/predefined-agent`
`POST /create/predefined-agent`